cmd/create/cluster/cmd.go

```diff
@@ -715,6 +714,11 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	if isHostedCP && r.Reporter.IsTerminal() {
+		r.Reporter.Infof("A billing account is needed but subscription" +
+			" charges will not be billed, Infrastructures costs still apply.")
```

Comment:
Should we add something here, that billing is not charged while in tech preview.

Reply:
Fixed, but this should not be merged for now